src/goto-instrument/cover.cpp

```diff
@@ -229,6 +229,10 @@ const char *as_string(coverage_criteriont c)
   case coverage_criteriont::PATH: return "path";
   case coverage_criteriont::MCDC: return "MC/DC";
   case coverage_criteriont::ASSERTION: return "assertion";
+  case coverage_criteriont::RUNTIME_EXCEPTIONS:
+    return "runtime_exceptions";
```

Comment:
The naming looks a bit inconsistent:  space seems to be used here instead of underscore.